### ISSUE TITLE
Replace AngularJS based test suite for CORS with something else

### ISSUE THREAD

**Issue by jonkoops** [keycloak/keycloak#24702]
We have [an application](https://github.com/keycloak/keycloak/tree/main/testsuite/integration-arquillian/test-apps/cors) to test CORS in the test suite that is still based on the now end-of-life AngularJS. This should be re-written into something that is actually still supported such as plain JavaScript, or something like [Alpine.js](https://alpinejs.dev/).

**Comment by nabeelnoor** [comment]
@jonkoops have anyone started work on this issue, or I can provide help on this one??

**Comment by jonkoops** [comment]
Hi @nabeelnoor, nobody has claimed this issue yet. So if you like I can assign it to you. I can make time available to guide and review. Let me know what you'd like to do.

**Comment by nabeelnoor** [comment]
> Hi @nabeelnoor, nobody has claimed this issue yet. So if you like I can assign it to you. I can make time available to guide and review. Let me know what you'd like to do.

Yes I like to work on this issue, plus @jonkoops your guidance on this is highly appreciated.

**Comment by sharoha** [comment]
Hi. I was going through this issue and it seems that there was not much progress on this. Can we reinitiate this issue, if not resolved yet?

**Comment by jonkoops** [comment]
Looks like in the mean time the Angular application was removed, so this issue is resolved. Thanks for letting me know.
